### PR TITLE
Fix typo in learn question answer.

### PR DESCRIPTION
**Pull request by leerob**
Surfaced through https://twitter.com/BrooksLybrand/status/1308450025706582018. My bad!

### CODE REVIEW

File: pages/learn/basics/data-fetching/pre-rendering.mdx [hunk 1]
```diff
@@ -63,7 +63,7 @@ Next, let’s talk about **two forms** of pre-rendering in Next.js.
       'Allows your application to work without JavaScript',
       'Allows your appplication to only run server-side'
```
Comment (chibicode) [inline pages/learn/basics/data-fetching/pre-rendering.mdx:64]:
```suggestion
      'Allows your application to only run server-side'
```

File: pages/learn/basics/data-fetching/pre-rendering.mdx [hunk 2]
```diff
@@ -63,7 +63,7 @@ Next, let’s talk about **two forms** of pre-rendering in Next.js.
...
     ]}
-  correctAnswer="Allows your appplication to run only client-side"
+  correctAnswer="Allows your appplication to only run server-side"
```
Comment (chibicode) [inline pages/learn/basics/data-fetching/pre-rendering.mdx:66]:
```suggestion
  correctAnswer="Allows your application to only run server-side"
```

**Review by chibicode — APPROVED** [review]

File: pages/learn/basics/data-fetching/pre-rendering.mdx (hunk 1 above)
Comment (leerob) [reply pages/learn/basics/data-fetching/pre-rendering.mdx]: Thank you! I was going to check the others after this. On top of it 😄

**Comment by leerob** [comment]
Thank you! 🙏

**Pull request merged**